cmd/manager/main.go

```diff
@@ -29,16 +32,38 @@ func printVersion() {
 	log.Info(fmt.Sprintf("Component version: %s", version.String))
 }
 
+func setupLogger(devLogging bool, logLevel string) {
+	// The logger instantiated here can be changed to any logger
+	// implementing the logr.Logger interface. This logger will
+	// be propagated through the whole operator, generating
+	// uniform and structured logs.
+	var level zap.AtomicLevel
+	switch logLevel {
+	case "debug":
+		level = zap.NewAtomicLevelAt(zap.DebugLevel)
+	case "info":
+		level = zap.NewAtomicLevelAt(zap.InfoLevel)
+	case "warn":
+		level = zap.NewAtomicLevelAt(zap.WarnLevel)
+	case "error":
+		level = zap.NewAtomicLevelAt(zap.ErrorLevel)
+	default:
+		level = zap.NewAtomicLevelAt(zap.InfoLevel)
```

Comment:
Should we log a warning that the log level given wasn't understood so we're defaulting to "info"?